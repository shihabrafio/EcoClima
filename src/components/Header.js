import React from 'react';

const Header = () => (
  <>
    <nav>
      <img className="img-nav" src="cloudy.png" alt="Logo" />
      <h1>ECOCLIMA</h1>
    </nav>
    <header>
      <div className="img-plh"><img src="asia.png" alt="Asia Map" /></div>
      <h1>Air Quality In Asian Countries.</h1>
    </header>
  </>
);

export default Header;
