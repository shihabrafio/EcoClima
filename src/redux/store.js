import { configureStore } from '@reduxjs/toolkit';
import CountryReducer from './countrySlice';

const store = configureStore({
  reducer: CountryReducer,
});

export default store;
