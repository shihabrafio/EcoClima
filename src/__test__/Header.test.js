import React from 'react';
import { render as rtlRender } from '@testing-library/react';
import { Provider } from 'react-redux';
import store from '../redux/store';
import Header from '../components/Header';

const render = (component) => rtlRender(
  <Provider store={store}>
    {component}
  </Provider>,
);

describe('Header', () => {
  it('render Header Components', () => {
    render(<Header />);
    expect(Header).toMatchSnapshot();
  });
});
