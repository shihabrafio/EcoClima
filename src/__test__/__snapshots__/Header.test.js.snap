// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Countires render Header Components 1`] = `[Function]`;

exports[`Header render Header Components 1`] = `[Function]`;
